common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signin/SignInTokenApiResponse.kt

```diff
@@ -46,9 +46,10 @@ data class SignInTokenApiResponse(
     @Expose @SerializedName("error") val error: String?,
     @Expose @SerializedName("error_description") val errorDescription: String?,
     @Expose @SerializedName("error_uri") val errorUri: String?,
-    @Expose @SerializedName("details") val details: List<Map<String, String>>?,
     @Expose @SerializedName("error_codes") val errorCodes: List<Int>?,
-    @Expose @SerializedName("inner_errors") val innerErrors: List<InnerError>?
+    @SerializedName("continuation_token") val continuationToken: String?,
+    @Expose @SerializedName("client_info") val clientInfo: String?,
```

Comment:
why `client_info` this re-introduced?

Reply:
it was wrongly introduced during merging from dev. Removed now